spark_board/html.py

```diff
@@ -24,15 +24,17 @@
 
 class Encoder(json.JSONEncoder):
     def default(self, o: Any) -> Any:
+        if type(o) is Condition:
```

Comment:
Why? It would only make a difference if we extend the class `Condition`, and in that case I don't know if we want to use the `dataclasses.asdict` for that new type

Reply:
it's the standard way of checking types in Python, but if you don't want to do it for subclasses I guess it's ok